preview/full/global.ini

```diff
@@ -50132,7 +50132,7 @@ interaction_restraints_unrestrain=Befreien
 interaction_sit=Setzen
 interaction_situp=Aufrecht sitzen
 interaction_sleep=Schlafen
-interaction_store=Einlagern
+interaction_store=Einstecken
```

Comment:
Kann man mal ausprobieren. Weiß noch nicht ob das nicht zu mehrdeutig ist. (Wenns an der Wand zb hängt könnte man denken dass man was reinstecken kann)

Reply:
Jo. Finde nur den englischen Begriff "Store" merkwürdig. Kenne es nicht in diesem Zusammenhang. Einstecken wäre perfekt wenn es tatsächlich nur das Stecken bedeutet